src/events/ready/daily-games.js

```diff
@@ -12,4 +12,4 @@ module.exports = async (c) => {
         const selectedGame = games[Math.floor(Math.random() * games.length)];
         selectedGame(c);
     });
-};
\ No newline at end of file
+    };
```

Comment:
The indentation for this is a little off, you could disregard the file if that's easier!

Reply:
See comment below